docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -13,6 +13,85 @@ ms.devlang: azurecli
 
 # Azure CLI release notes
 
+## August 13, 2019
+
+Version 2.0.71
+
+### AppService
+
+* az webapp webjob continuous group commands were failing for slots
+
+### BotService
+
+* BREAKING CHANGE:
+    create:
+        - Removed support for creating v3 SDK bots
+        - Remove `az bot publish` example when creating a Web App bot
+
+### CognitiveServices
+
+* Add "cognitiveservices account network-rule" commands.
+
+### Cosmos DB
+
+* Remove warning when updating multiple write locations
+* Add CRUD commands for CosmosDB SQL, MongoDB, Cassandra, Gremlin and Table resources and resource's throughput.
+
+### HDInsight
+
+* BREAKING CHANGE:
+    - create:
+        - Renamed --storage-default-container to --storage-container and --storage-default-filesystem to --storage-filesystem
+    - application create:
+        - Changed the --name/-n argument to represent the application name instead of the cluster name and added a separate --cluster-name argument
+        - Renamed --application-type to --type/-t
+        - Renamed --marketplace-identifier to --marketplace-id
+        - Renamed --https-endpoint-access-mode to --access-mode and --https-endpoint-destination-port to --destination-port
+        - Removed --https-endpoint-location, --https-endpoint-public-port, --ssh-endpoint-destination-port, --ssh-endpoint-location and --ssh-endpoint-public-port
+    - resize:
+        - Renamed --target-instance-count to --workernode-count/-c
+    - script-action
```

Comment:
@yugangw-msft  This is a command group, not a command. Do these notes apply to all commands in the group?

Reply:
Correct